pkg/scheduler/core/equivalence_cache_test.go

```diff
@@ -342,40 +342,42 @@ func TestUpdateResult(t *testing.T) {
 		},
 	}
 	for _, test := range tests {
-		ecache := NewEquivalenceCache()
-		if test.expectPredicateMap {
-			ecache.algorithmCache[test.nodeName] = AlgorithmCache{}
-			predicateItem := HostPredicate{
-				Fit: true,
-			}
-			ecache.algorithmCache[test.nodeName][test.predicateKey] =
-				PredicateMap{
-					test.equivalenceHash: predicateItem,
+		t.Run(test.name, func(t *testing.T) {
+			ecache := NewEquivalenceCache()
+			if test.expectPredicateMap {
+				ecache.algorithmCache[test.nodeName] = AlgorithmCache{}
+				predicateItem := HostPredicate{
+					Fit: true,
 				}
-		}
+				ecache.algorithmCache[test.nodeName][test.predicateKey] =
+					PredicateMap{
+						test.equivalenceHash: predicateItem,
+					}
+			}
 
-		node := schedulercache.NewNodeInfo()
-		node.SetNode(&v1.Node{ObjectMeta: metav1.ObjectMeta{Name: test.nodeName}})
-		ecache.updateResult(
-			test.pod,
-			test.predicateKey,
-			test.fit,
-			test.reasons,
-			test.equivalenceHash,
-			test.cache,
-			node,
-		)
-
-		cachedMapItem, ok := ecache.algorithmCache[test.nodeName][test.predicateKey]
-		if !ok {
-			t.Errorf("Failed: %s, can't find expected cache item: %v",
-				test.name, test.expectCacheItem)
-		} else {
-			if !reflect.DeepEqual(cachedMapItem[test.equivalenceHash], test.expectCacheItem) {
-				t.Errorf("Failed: %s, expected cached item: %v, but got: %v",
-					test.name, test.expectCacheItem, cachedMapItem[test.equivalenceHash])
+			node := schedulercache.NewNodeInfo()
+			node.SetNode(&v1.Node{ObjectMeta: metav1.ObjectMeta{Name: test.nodeName}})
+			ecache.updateResult(
+				test.pod,
+				test.predicateKey,
+				test.fit,
+				test.reasons,
+				test.equivalenceHash,
+				test.cache,
+				node,
+			)
+
+			cachedMapItem, ok := ecache.algorithmCache[test.nodeName][test.predicateKey]
+			if !ok {
+				t.Errorf("Failed: %s, can't find expected cache item: %v",
```

Comment:
Remove test.name  and "Failed" from log messages. I think you had done this in a previous version, and accidentally reverted it?